### ISSUE TITLE
UX flow of generation

### ISSUE THREAD

**Issue by stevepeak** [microservices/omg.py#5]
When reviewing the README. I believe there is a missing step.

```
`If not available on disk, the required Dockerfile and microservice.yml files will automatically be generated, for your application:
```

How do you generate the `microservice.yml`? Do you call something via a CLI?

**Comment by kenneth-reitz** [comment]
that was the original design, but now you just call `service.serve(ensure=True)`. Room for improvement, but I think this is a good approach, for now. Perhaps going back to the CLI approach would be best.

**Comment by kenneth-reitz** [comment]
yes, i like the idea of moving back to the cli approach for the generation. `omg-ensure`.
